docs/usage/authentication.md

```diff
@@ -0,0 +1,44 @@
+There are several ways to authenticate using the SDK:
+
+```python
+from pasqal_cloud import SDK
+
+project_id = "your_project_id"  # Replace this value with your project_id on the PASQAL platform. It can be found on the user-portal, in the 'project' section.
+username = "your_username"  # Replace this value with your username or email on the PASQAL platform.
+password = "your_password"  # Replace this value with your password on the PASQAL platform.
+```
+
+## Method 1: Username + Password
+
+If you know your credentials, you can pass them to the SDK instance on creation:
```

Comment:
"If you know your credentials" seems out of place, there is no alternative anyways. 

```suggestion
Authenticate with the SDK with your email and password directly
```